### PR TITLE
BIP 39: Аdd swift library with multi lang support

### PR DESCRIPTION
**Pull request by skywinder**
Add one more swift library to generate mnemonics (with multi lang support)
Here is an implementation:
https://github.com/matter-labs/web3swift/blob/e36e767eb2de26decb0c157aeef2d023a1175076/Sources/web3swift/KeystoreManager/BIP39.swift#L72

### CODE REVIEW

**Comment by skywinder** [comment]
@luke-jr please, kindly check this small addition :wink:

**Comment by skywinder** [comment]
We moved our repo to the new address. I updated URL. Please, kindly check. 👍

**Comment by prusnak** [comment]
NAK. The URL points to the library that does other 1000 things than BIP39.

Maybe if the URL pointed to the relevant file ...

File: bip-0039.mediawiki [hunk 1]
```diff
@@ -165,6 +165,7 @@ Rust:
 Swift:
 * https://github.com/CikeQiu/CKMnemonic
 * https://github.com/yuzushioh/WalletKit
```
Comment (skywinder) [inline bip-0039.mediawiki:167]: (Would like to mention, that other links doesn't direect to the class itself)

**Comment by skywinder** [comment]
> Maybe if the URL pointed to the relevant file ...

@luke-jr @prusnak can you reopen this PR, so I will update the link to the corresponding class? (or should I make another PR with the correct link? i.e. [web3swift/KeystoreManager/BIP39.swift](https://github.com/matter-labs/web3swift/blob/e36e767eb2de26decb0c157aeef2d023a1175076/Sources/web3swift/KeystoreManager/BIP39.swift)

**Comment by skywinder** [comment]
Done, @luke-jr

File: bip-0039.mediawiki (hunk 1 above)
Comment (prusnak) [reply bip-0039.mediawiki:167]: They include the example of how generate a mnemonic right on top of their readme.

File: bip-0039.mediawiki [hunk 2]
```diff
@@ -165,7 +165,8 @@ Rust:
...
-* https://github.com/matter-labs/web3swift (`BIP39` public class)
+* https://github.com/matter-labs/web3swift/blob/develop/Sources/web3swift/KeystoreManager/BIP39.swift
+* https://github.com/matter-labs/web3swift/blob/develop/Sources/web3swift/KeystoreManager/BIP39.swift
```
Comment (prusnak) [inline bip-0039.mediawiki]: Once is enough

**Pull request merged**